python/pyspark/sql/functions.py

```diff
@@ -187,6 +187,15 @@ def _():
                        ' column name, and null values appear after non-null values',
 }
 
+_functions_3_0_over_column = {
+    'asinh': """:param col: hyperbolic angle
```

Comment:
Can we avoid to add them per https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/functions.scala#L42-L56?